CodingChallenges/CC_052_random_walk/Processing/CC_052_random_walk/CC_052_random_walk.pde

```diff
@@ -0,0 +1,38 @@
+// Daniel Shiffman
+// http://codingtra.in
+// http://patreon.com/codingtrain
+// Video: https://youtu.be/l__fEY1xanY
+// Processing transcription: Chuck England
+
+int x;
+int y;
+
+void setup() {
+  size(400, 400);
+  x = 200;
+  y = 200;
+  background(51);
+}
+
+void draw() {
+  stroke(255, 100);
+  strokeWeight(2);
+  point(x, y);
+
+  int r = floor(random(4));
```

Comment:
```suggestion
  int r = int(random(4));
```

Tiniest of points but Processing has an `int()` method to convert to integer which is more typical of examples (I think?)